src/dcegm/solve.py

```diff
@@ -109,8 +110,18 @@ def solve_dcegm(
         compute_utility=compute_utility,
         compute_marginal_utility=compute_marginal_utility,
     )
+    (
+        state_choices_per_period,
+        states_per_period,
+    ) = determine_states_and_state_choices_per_period(
+        state_space=state_space,
+        state_choice_space=state_choice_space,
+        num_periods=n_periods,
+    )
 
     backwards_induction(
+        state_choices_per_period=state_choices_per_period,
```

Comment:
state_choice_combs_per_period

Reply:
Let us abandon combs in future. They are state choices